src/streamlink_cli/argparser.py

```diff
@@ -467,6 +467,15 @@ def boolean(value):
 
 stream = parser.add_argument_group("Stream options")
 stream.add_argument(
+    "--url",
+    dest="url_param",
+    metavar="URL",
+    help="""
+    Alternative parameter to specify a URL to attempt to
```

Comment:
Is there a way we could make this more user friendly in terms of explanation? I feel like it is a bit confusing, not sure if there is a better way though.